src/engine-actions/translate-swap.js

```diff
@@ -1,220 +1,59 @@
-      let clientCancel = false
-
-      const errorListener = (err = new Error('Unknown LND Error')) => {
-        // if we cancelled the request, we treat a CANCELLED code
-        // as a cleanup and not a true error
-        if (clientCancel && err.code && err.code === CANCELLED_CODE) {
-          logger.debug('Received CANCELLED after client cancelled, cleaning up')
-          finish()
-        } else {
-          logger.error(`Error from sendToRoute stream`, err)
-          finish(err)
-        }
-      }
-
-      const endListener = () => {
-        logger.error(`LND closed sendToRoute stream before returning a value`)
-        return finish(new Error(`LND closed stream to send to route`))
-      }
-
-      const dataListener = (response) => {
-        const { paymentError, paymentPreimage } = response
-        logger.debug('Received sendToRoute response from LND', { paymentError, paymentPreimage })
-
-        // cancel will trigger cleanup through our errorListener
-        clientCancel = true
-        stream.cancel()
-
-        // TODO: do we need to ensure that this preimage matches the hash that we sent?
-        resolve({ paymentError, paymentPreimage })
-      }
-
-      // Helper to make sure we tear down our listeners
-      const finish = (err, response) => {
-        stream.removeListener('error', errorListener)
-        stream.removeListener('end', endListener)
-        stream.removeListener('data', dataListener)
-
-        if (err) {
-          return reject(err)
-        }
-      }
-
-      stream.on('error', errorListener)
-      stream.on('end', endListener)
-      stream.on('data', dataListener)
-
-      logger.debug('Writing to sendToRoute stream', { paymentHash })
-      stream.write({ paymentHash, routes })
-    } catch (e) {
-      reject(e)
-    }
-  })
-}
-
 /**
  * @typedef {Object} TranslateSwapOutcome
- * @property {String} paymentPreimage Base64 string of the preimage for the swapHash
- * @property {String} permanentError Error encountered that is permanent, and safe to cancel the upstream HTLC
+ * @property {String} paymentPreimage Base64 string of the preimage for the
+ *                                    swapHash
+ * @property {String} permanentError Error encountered that is permanent, and
+ *                                   safe to cancel the upstream HTLC
  */
 
 /**
  * Translates a swap to a new payment channel network by making a payment
  * to the counterparty (Taker) node on this network
  *
- * @param {string} takerAddress - Payment channel network address fo the Taker node we are paying
- * @param {string} swapHash     - swap hash that will be associated with the swap
- * @param {string} amount       - Int64 string of the amount of outbound currency in integer units
- * @param {string} extendedTimeLockDelta  - Int64 string of the time lock extended to us by on the first chain in seconds
+ * @param {string} takerAddress          - Payment channel network address of
```

Comment:
@treygriffith is this a tool/formatting change for jsdoc stuff or are you doing this to break the comments up? I'd be curious to know if this actually parses correctly

Reply:
I was trying to keep my lines shorter (80 characters). How can I check the parsing?

Reply:
we can worry about when the time comes. no worries